### PR TITLE
Fix adaptive expressions divide function

### LINKED ISSUE #3527: Adaptive Expressions 'divide' with float is not returning a float
## Versions
What package version of the SDK are you using. **4.12.0**
What nodejs version are you using **14.7.0**

## Describe the bug
Adaptive Expressions, divide is not working as expected nor as documented. In the documentation, if one of the arguments is a float, a float is expected to be returned. See docs here: https://docs.microsoft.com/en-us/azure/bot-service/adaptive-expressions/adaptive-expressions-prebuilt-functions?view=azure-bot-service-4.0#div

and screenshot of it: divide docs screenshot

## To Reproduce
1. try expression: `div(11.2, 2)`

## Expected behavior
returns `5` instead of `5.6`
also, the docs have a funny example. `div(11.2, 3)` does not equal `5.5` :)

## Additional context
also doesn't work for `11.2 / 2`

### PR DESCRIPTION
**Pull request by stefangomez**
Fixes #3527 

## Description
Corrects the expected result for floats in a divide expression

## Specific Changes
  -  changed the return value for Divide to only do a Math.floor if both args are Integers

## Testing
  - added tests for `div(11.2, 2)` and `11.2 / 2`

### CODE REVIEW

File: libraries/adaptive-expressions/src/builtinFunctions/divide.ts [hunk 1]
```diff
@@ -26,7 +26,11 @@ export class Divide extends MultivariateNumericEvaluator {
      * @private
      */
     private static func(args: any[]): number {
-        return Math.floor(Number(args[0]) / Number(args[1]));
+        let result: number = Number(args[0]) / Number(args[1]);
```
Comment (Danieladu) [inline libraries/adaptive-expressions/src/builtinFunctions/divide.ts:29]: `let` -> `const`

**Comment by Danieladu** [comment]
LGTM

File: libraries/adaptive-expressions/src/builtinFunctions/divide.ts (hunk 1 above)
Comment (stefangomez) [reply libraries/adaptive-expressions/src/builtinFunctions/divide.ts:29]: Ah my bad, I was trying to follow convention I saw in other files, but I see that 'let' was primarily only used for actual mutable vars. I see const other places accordingly. pushing the change now

**Review by stevengum — APPROVED** [review]

**Comment by stevengum** [comment]
@stefangomez thank you for filing the issue and creating this PR!

**Pull request merged**